packages/compass-export-to-language/src/typings.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'bson-transpilers';
+declare module 'js-beautify';
```

Comment:
Looks like there's a typings library for this we might be able to use:
https://www.npmjs.com/package/@types/js-beautify